src/App.tsx

```diff
@@ -0,0 +1,36 @@
+import { BrowserRouter, Route, Switch } from "react-router-dom";
+import { FunctionComponent } from "react";
+import Navbar from "./components/Navbar";
+import About from "./containers/About";
+import Feed from "./containers/Feed";
+import Profile from "./containers/Profile";
+import Login from "./containers/Login";
+import Signup from "./containers/Signup";
+import Home from "./containers/Home";
+import Notification from "./containers/Notification";
+
+
+
+const App:FunctionComponent = () => {
+  return (
+    <div>
+      <BrowserRouter>
+      <Navbar />
+        <Switch>
+          <Route path={" About, Feed, Profile, Signup, Login, Notification , /"} />         
```

Comment:
This Route is a self closing tag which is not right, this one should be the parent to every other Route tag inside this Switch and also the Layout component should also be in this Route tag

Reply:
Thanks for your review. Working on it